catalog/tests/dags/providers/provider_api_scripts/test_flickr.py

```diff
@@ -176,17 +176,26 @@ def test_get_record_data():
     assert actual_data == expected_data
 
 
-def test_get_record_data_returns_none_when_missing_owner():
-    image_data = _get_resource_json("image_data_complete_example.json")
-    image_data.pop("owner")
-
+@pytest.mark.parametrize(
+    "image_data",
+    [
+        {"license_info": None},
+        {"license_info": ""},
+        {"url_l": None},
+        {"url_l": ""},
+        {"id": None},
+        {"id": ""},
+    ],
+)
+def test_get_record_data_returns_none_when_required_params_falsy(image_data):
```

Comment:
Won't these tests all fail immediately at the `license_info` check? So we're not really testing the falsy url/id. Could we make the test similar to `test_get_record_data_returns_none_when_missing_owner` below, where we build an otherwise complete `image_data` and then modify it to make the value under test falsy?

Reply:
Great catch, you're right, @stacimc . I'll update the tests